lib/components/admin/call-record.js

```diff
@@ -47,29 +57,42 @@ export default class CallRecord extends Component {
             In progress... click <Icon type='stop' /> to save{' '}
             ({call.searches.length} searches)
           </small>
-          <div>
+          <ul className='list-unstyled'>
             {activeQueries.length > 0
               ? activeQueries.map((query, i) => (
                 <QueryRecord key={i} query={query} index={i} />
               ))
               : 'No queries recorded.'
             }
-          </div>
+          </ul>
         </div>
       )
     }
+    // Default (no active call) view
+    const startTimeMoment = moment(call.startTime)
     return (
-      <div style={{margin: '5px 0'}}>
-        <button
-          style={{width: '100%'}}
+      <div style={{borderBottom: '1px solid grey', margin: '5px 0'}}>
+        <CallRecordButton
           className='clear-button-formatting'
           onClick={this._toggleExpanded}
         >
-          <Icon type='phone' className='fa-flip-horizontal' />
-          Call {index} ({moment(call.endTime).fromNow()})
-        </button>
+          <CallRecordIcon
+            className='fa-flip-horizontal'
+            style={{width: '5%'}}
+            type='phone' />
+          <span style={{flex: '0 0 120px'}}>
+            {startTimeMoment.format('h:mm a, MMM D')}
```

Comment:
(Nitpick: the minutes portion of the time should ideally be flushed right... non blocking)